install/kubernetes/cilium/templates/cilium-agent/rolebinding.yaml

```diff
@@ -52,6 +50,6 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ .Values.serviceAccounts.cilium.name | quote }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ template "cilium.namespace" . }}
 {{- end}}
-
+{{- end -}}
```

Comment:
I suppose that moving this `{{- end -}}` to the bottom of the file is the lint issue that you mention in the commit description? I'd recommend moving it to a separate commit, for clarity.

Reply:
- [x] A separate commit for this